Source/PopTip.swift

```diff
@@ -691,7 +691,14 @@ open class PopTip: UIView {
     containerView = view
     let controller = UIHostingController(rootView: rootView)
     controller.view.backgroundColor = .clear
-    let maxContentWidth = UIScreen.main.bounds.width - (self.edgeMargin * 2) - self.edgeInsets.horizontal - (self.padding * 2)
+    let window = parent.view.window
```

Comment:
I think you can remove this declaration since you safely unwrap `parent.view.window` at line 696

Reply:
Good catch, fixed.